lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/or/rest/SemanticWiki.java

```diff
@@ -0,0 +1,120 @@
+     * 	       <uri>transfer:obj.34872</uri>  	<!-- uri of ontology instance-->
+     *         <description></description>
+     *         <name>Sally Shugar</name>
+     *         <email>sally.shugar@learnpad.eu</email>
+     *         <skypeId>sshugar@learnpad.eu</skypeId>
+     *         <phoneNumber>+39 0733 656 344</phoneNumber>
+     *         <officeAddress>Curved street 22A</officeAddress>
+     *         <role>SUAP Officer</role>
+     *         <organisationalUnit>
+     *           <name>SUAP Office</name>
+     *           <uri>transfer:obj.35099</uri> <!-- uri of ontology instance-->
+     *         </organisationalUnit>
+     *     </businessActor>
+     *     <relatedObjects>
+     *       <relatedObject>
+     *         <relationType>sameAuthor</relationType> <!-- [sameCreator|sameAuthor|sameTaskContext|page] -->
+     *         <uri>transfer:obj.34748</uri>
+     *         <name>Pioneering Sustainable Buildings</name>
+     *         <documentUrl>https://www.youtube.com/watch?v=lwsEV-lDqmo</documentUrl>
+     *         <mimeType>text/html</mimeType>
+     *         <description>Example of applying evolving water and power technologies with new light management and computer controlled environment</description>
+     *         <comment></comment>
+     *       </relatedObject>
+     *     </relatedObjects>
+     *   </entity>
+     * </entities>
+     * 
+     * </code>
+     * 
+     * @param modelSetId
+     * @param contextArtifactId
+     * @param userId
+     * @param title
+     * @param text (HTML) text to analyse and anotate. 
+     * @return
+     * @throws LpRestException 
+     */
+    @Path("/{modelsetid}/analysetext")
+    @POST    
+    Entities analyseText(@PathParam("modelsetid") String modelSetId,
+            @QueryParam("contextArtifactId") String contextArtifactId,
+            @QueryParam("userid") String userId, 
+            @QueryParam("title") String title, 
+            String text) throws LpRestException;
+    
+    /**
+     * Creates a bookmark of a selected named entity (person, document, organisationalUnit).
+     * Either a global bookmark or one for the context is created. In case of a context realted 
+     * bookmark, the artifactId (page id or URI of ontology instance, like a task) is required and considered as context. 
+     * 
+     * @param modelSetId
+     * @param userId
+     * @param artifactId the URI of the recognized entity resp. resource (person, document, OU) to bookmark
+     * @param contextArtifactId the id of the context (page or URI of ontology instance)
+     * @throws LpRestException 
+     */
+    @Path("/{modelsetid}/bookmark")
+    @GET
+    void createBookmark(@PathParam("modelsetid") String modelSetId,
+            @QueryParam("userid") String userId, 
+            @QueryParam("artifactId") String artifactId,
+            @QueryParam("contextArtifactId") String contextArtifactId) throws LpRestException;    
```

Comment:
Do I understand properly that we want for you to be able to bookmark any text which is highlighted by the text analysis in any comment but not anything else?